test/suite/test_backup15.py

```diff
@@ -113,4 +113,3 @@ def test_backup15(self):
                 self.take_full_backup(self.home_full)
             self.compare_backups(self.uri, self.home_full, self.home_incr, str(self.bkup_id))
             self.setup_directories(self.home_incr, self.home_full)
```

Comment:
Think you need to add back in 
```
if __name__ == '__main__':
    wttest.run()
```
at the end of the test.

Reply:
Why? I don't think this is useful. Do we run these test from main?

Reply:
I believe we don't usually run these tests from main. But just as a coding style convention, from my understanding we add it to each test. I could definitely be wrong here, then don't worry about adding that piece of code.